assemble/conf/accumulo.properties

```diff
@@ -31,3 +31,19 @@ instance.secret=DEFAULT
 
 ## Set to false if 'accumulo-util build-native' fails
 tserver.memory.maps.native.enabled=true
+
+## Set compaction settings
+### Default Compactor
+compaction.service.default.planner=org.apache.accumulo.core.spi.compaction.DefaultCompactionPlanner
+compaction.service.default.planner.opts.maxOpen=10
+compaction.service.default.planner.opts.executors="[{'name':'small','type':'internal','maxSize':'32M','numThreads':2},{'name':'medium','type':'internal','maxSize':'128M','numThreads':2},{'name':'large','type':'internal','numThreads':2}]"
+
+### Meta Compactor
```

Comment:
This comment is just for discussion, not recommending any change.   Wondering if the new example props in Property.java could be placed in this file via a generation step.  An advantage of that is that it lowers the maintenance overhead.  Not sure what the disadvantages are though.